eclair-core/src/test/resources/nonreg/codecs/000003-DATA_NORMAL/funder/data.json

```diff
@@ -111,8 +111,7 @@
     "remotePerCommitmentSecrets" : null,
     "originChannels" : { }
   },
-  "shortIds" : {
-    "real" : "1413373x969x0",
+  "aliases" : {
```

Comment:
It's a shame that `localFundingStatus` is unconfirmed in these test data, we cannot test the setting of real scid.